### ISSUE TITLE
mysql.server: command not found (MySQL 5.7.10/11)

### ISSUE THREAD

**Issue by lukejanicke** [pbougie/mac-dev-env#16]
I followed your instructions exactly but updated the version to 5.7.11.

> ### New Installation
> Initialize your database server.
> 
> `mysqld --initialize --basedir=/usr/local/mysql --datadir=/usr/local/var/mysql`

When I issue this command, I get some errors and then I can't login.

````
~ $ mysqld --initialize --basedir=/usr/local/mysql --datadir=/usr/local/var/mysql
2016-03-02T14:54:07.416323Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
2016-03-02T14:54:07.428807Z 0 [Warning] Setting lower_case_table_names=2 because file system for /usr/local/var/mysql/ is case insensitive
2016-03-02T14:54:07.714710Z 0 [Warning] InnoDB: New log files created, LSN=45790
2016-03-02T14:54:07.750527Z 0 [Warning] InnoDB: Creating foreign key constraint system tables.
2016-03-02T14:54:07.815327Z 0 [Warning] No existing UUID has been found, so we assume that this is the first time that this server has been started. Generating a new UUID: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.
2016-03-02T14:54:07.834034Z 0 [Warning] Gtid table is not ready to be used. Table 'mysql.gtid_executed' cannot be opened.
2016-03-02T14:54:07.841333Z 1 [Note] A temporary password is generated for root@localhost: XXXXXXXXXXXX
~ $ mysql -u root -p
-bash: mysql.server: command not found
~ $
````

I'm using OS X El Capitan Version 10.11.3.

**Comment by lukejanicke** [comment]
Tried again with MySQL 5.7.10. Same issue. I must be missing a step.

Only differences, I used the newer Git 2.7.0 (instead 2.6.4) and OpenSSL 1.0.2e (which has since been updated to 1.0.2f on your site, and 1.0.2g on the OpenSSL site).

**Comment by lukejanicke** [comment]
````
~ $ echo $PATH
/usr/local/mysql/bin:/usr/local/cmake/bin:/usr/local/git/bin:/usr/local/openssl/bin:/usr/local/autoconf/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
~ $
```` 

PATH is correct.

**Comment by lukejanicke** [comment]
I found an old alias in `~/.profile` for `mysql`. I deleted that.

I got rid of the timestamp warning by changing the command to:

`mysqld --initialize --basedir=/usr/local/mysql --datadir=/usr/local/var/mysql --explicit_defaults_for_timestamp=1`

However, now, upon running `mysql -u root -p`, I get the following warning.

`ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/tmp/mysql.sock' (2)`

That is a different issue. So, I'm closing this issue and opening another.
